notebooks/official/feature_store/online_feature_serving_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    FEATURE_VIEW_ID = "my_feature_view"  # @param {type: "string"}
</pre><p>nit: default ID: "feature_view_publications", instead of "my_feature_view"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Fixed</p>